docs/UserGuide.md

```diff
@@ -342,9 +357,31 @@ Exits the program.
 
 Format: `exit`
 
-### 6.11. Viewing daily tips
+### 6.11. Viewing application stage tips
```

Comment:
need to think where's the best place for this section, but definitely not after `exit` because thats literally at the end already...
![Screenshot 2022-11-02 at 1 18 09 AM](https://user-images.githubusercontent.com/77665182/199295851-eae1b96d-6c62-436f-b22f-b28da402d393.png)

Reply:
I moved it so its right below the list command because after users add and list the applications, they can view the tips.